README.md

```diff
@@ -22,41 +18,34 @@ SatSale is a lightweight Bitcoin payment processor that connects to your own Bit
 - [Updating](#Updating)
 - [Docs](#Docs)
 - [Contributions welcomed](#contributions-welcomed)
-- [Coming soon](#coming-soon)
 - [Disclaimer](#disclaimer)
 - [Support](#support)
 
 ![logo](static/logo.svg)
 
-# Purpose
-
-SatSale currently serves as a
+Satsale can be used as a
 
 1. Donation page and button for your website that you can easily embed/link to anywhere.
 2. Bitcoin payment gateway, including a Woocommerce plugin that easily turns any Wordpress site into a Bitcoin accepting store.
 3. Versatile API and payments platform for both on-chain and lightning payments (supporting both clightning and lnd).
 
-While other Bitcoin payment processors are known for being difficult to install and self-host, SatSale is easy to modify and build upon.
+Other Bitcoin payment processors is small, easy to install and self-host. SatSale is also a great building block for python lightning applications.
```

Comment:
Don't get this change. You are now saying that other payment processors is small and easy to install and self-host, but not SatSale?

Reply:
thanks!